lerna.json

```diff
@@ -18,7 +18,6 @@
   ],
   "version": "2.0.0-rc.3",
   "npmClient": "yarn",
```

Comment:
Is it the case that with lerna we do not need this configuration option (as vs lerna lite)?

Reply:
@RafaelAPB Good question, I'm not sure TBH. It could be that lerna ignores this option, but I didn't look up their docs. Maybe @jenniferlianne knows.